lib/Controller/Campaign.php

```diff
@@ -144,6 +144,16 @@ public function grid()
                 'text' => __('Schedule Now')
             );
 
+            // Preview
+            // // FIXME : check permissions pour ce bouton 
```

Comment:
I don't think you have anything to fix here - we wouldn't be in the loop unless we had view permissions for the Campaign in question.